platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Style.java

```diff
@@ -296,21 +300,32 @@ public void addImage(@NonNull String name, @NonNull Bitmap image) {
   /**
    * Adds an image to be used in the map's style
    *
-   * @param name  the name of the image
-   * @param image the pre-multiplied Bitmap
-   * @param sdf   the flag indicating image is an SDF or template image
+   * @param name   the name of the image
+   * @param bitmap the pre-multiplied Bitmap
+   * @param sdf    the flag indicating image is an SDF or template image
    */
-  public void addImage(@NonNull String name, @NonNull Bitmap image, boolean sdf) {
+  public void addImage(@NonNull final String name, @NonNull final Bitmap bitmap, boolean sdf) {
     validateState("addImage");
-    nativeMapView.addImage(name, image, sdf);
+    new BitmapImageConversionTask(nativeMap, sdf).execute(new HashMap<String, Bitmap>() {
```

Comment:
This creates an unnecessary subclass instance of the `HashMap`. Could you lift the initialization up?